src/hdmf/common/multi.py

```diff
@@ -0,0 +1,23 @@
+from ..container import Container, Data, MultiContainerInterface
+from ..utils import docval, call_docval_func, popargs
+
+from . import register_class
+
+
+@register_class('SimpleMultiContainer')
+class SimpleMultiContainer(MultiContainerInterface):
+
+    __clsconf__ = {
+        'attr': 'containers',
+        'type': (Container, Data),
+        'add': 'add_container',
+        'get': 'get_container',
+    }
+
+    @docval({'name': 'name', 'type': str, 'doc': 'the name of this container'},
+            {'name': 'containers', 'type': (list, tuple), 'default': None,
+             'doc': 'the Container or Data objects in this file'})
+    def __init__(self, **kwargs):
+        containers = popargs('containers', kwargs)
+        call_docval_func(super().__init__, kwargs)
+        self.containers = containers
```

Comment:
It looks like this `__init__` does not do anything different from the default `__init__` generated by `MultiContainerInterface`. Could it be removed?

Reply:
The autogenerated one flips the order--`containers` is the first argument, `name` is the second. I prefer it the other way around

Reply:
OK